UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -259,7 +259,7 @@ public TemperatureAlert ExposeTemperature(float environmentalTemperature)
 				if (environmentalTemperature < min)
 				{
 					//so, Half Temperature of the minimum threshold that's when the maximum damage will kick in
-					TakeDamage(null,   0.25f*Mathf.Clamp((min-environmentalTemperature)/(min/2f), 0f,1f), AttackType.Internal, DamageType.Burn);
+					TakeDamage(null,   1.5f*Mathf.Clamp((min-environmentalTemperature)/(min/2f), 0f,1f), AttackType.Internal, DamageType.Burn, true);
```

Comment:
since you are using the same number in 3 different blocks of code, move it to a constant of the class. That would also help identify what is the number suppsoed to be.

I would also like to know what makes you change the number this time.

Reply:
random balancing numberm, is because change from damage each body part with X amount of damage , because it breaks down if you have only one body part, to damage split so therefore you have to multiply by the number of average body parts the person has, to get the balanced value

Reply:
uhm. I still think you should move it to a constant so it only needs to change once any time balance calls for it. Name it ``DAMAGE_FORMULA_FACTOR`` or something like that, I guess.